roles.yml

```
@@ -1,3 +1,3 @@
 ---
 - src: gforcada.compile_python
-  version: 6.0.0
+  version: 6.4.0
```

```suggestion
  version: 6.5.0
```

This new version already solves the unknown dependency.